x-pack/plugins/security_solution/public/common/components/event_details/overview/index.tsx

```diff
@@ -0,0 +1,178 @@
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const severityData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.severity', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const riskScoreData = useMemo(() => {
+      const item = find({ field: 'kibana.alert.rule.risk_score', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    const ruleNameData = useMemo(() => {
+      const item = find({ field: SIGNAL_RULE_NAME_FIELD_NAME, category: 'kibana' }, data);
+      const linkValueField = find({ field: 'kibana.alert.rule.uuid', category: 'kibana' }, data);
+      return (
+        item &&
+        getEnrichedFieldInfo({
+          eventId,
+          contextId,
+          timelineId,
+          browserFields,
+          item,
+          linkValueField,
+        })
+      );
+    }, [browserFields, contextId, data, eventId, timelineId]);
+
+    return (
+      <NotGrowingFlexGroup gutterSize="s">
+        {hasData(statusData) && (
+          <EuiFlexItem>
+            <OverviewCard title={SIGNAL_STATUS}>
+              <StatusPopoverButton
+                eventId={eventId}
+                contextId={contextId}
+                enrichedFieldInfo={statusData}
+                indexName={indexName}
+                timelineId={timelineId}
+                handleOnEventClosed={handleOnEventClosed}
+              />
+            </OverviewCard>
+          </EuiFlexItem>
+        )}
+
+        {hasData(severityData) && (
+          <EuiFlexItem>
+            <OverviewCardWithActions
+              title={ALERTS_HEADERS_SEVERITY}
+              enrichedFieldInfo={severityData}
+              contextId={contextId}
+            >
+              <SeverityBadge value={severityData.values[0]} />
+            </OverviewCardWithActions>
+          </EuiFlexItem>
+        )}
+
+        {hasData(riskScoreData) && (
+          <EuiFlexItem>
+            <OverviewCardWithActions
+              title={ALERTS_HEADERS_RISK_SCORE}
+              enrichedFieldInfo={riskScoreData}
+              contextId={contextId}
+            >
+              {riskScoreData.values[0]}
+            </OverviewCardWithActions>
+          </EuiFlexItem>
+        )}
+
+        {hasData(ruleNameData) && (
+          <EuiFlexItem>
+            <OverviewCardWithActions
```

Comment:
The `Rule` card `Filter out` and `...` overflow actions are hidden when the browser is resized per the screenshot below:

![rule_actions](https://user-images.githubusercontent.com/4459398/144666763-3d25d984-8ccc-4da0-82c9-94dea094888a.png)

Reply:
Great find, I adressed those issues in https://github.com/elastic/kibana/pull/120347/commits/d7da39d5bbb7c07b10ea30ce8235f2fd5ccab559 and https://github.com/elastic/kibana/pull/120347/commits/d33a0ab0e0177de8feb1aff9d18d0ef4c5c86d37. Also check out my comment explaining the new solution: https://github.com/elastic/kibana/pull/120347#issuecomment-987918713